packages/cli/src/modules/insights/database/repositories/insights-by-period.repository.ts

```diff
@@ -90,15 +90,18 @@ export class InsightsByPeriodRepository extends Repository<InsightsByPeriod> {
 		// SQLite by default
 		let periodStartExpr =
 			periodUnitToCompactInto === 'week'
-				? "strftime('%Y-%m-%d 00:00:00.000', date(periodStart, 'weekday 0', '-6 days'))"
+				? "datetime(date(periodStart, '-6 days', 'weekday 1'))"
```

Comment:
I like this, but it changes the returned format from `2025-05-26 00:00:00.000` to `2025-05-26 00:00:00`. You're sure there is no side effect of this ?
Especially since it's not the same format as the next line:
: `strftime('%Y-%m-%d ${periodUnitToCompactInto === 'hour' ? '%H' : '00'}:00:00.000', periodStart)`;